Libplanet/Consensus/VoteSet.cs

```diff
@@ -41,7 +41,7 @@ public class VoteSet
                 .Select(x => new VoteMetadata(
                     height,
                     round,
-                    blockHash,
+                    null,
```

Comment:
Could you explain why did you make this change?

Reply:
Initially, I thought it'd save space. On second thought, I think the requirement might have to be other way around?

Reply:
Did a complete U-turn and now requires `BlockHash` for `Vote`s with `Null` or `Unknown` flag. 😅